File: mbrs/src/main/java/com/ftn/mbrs/service/VoziloService.java
//Fri May 11 23:51:24 CEST 2018
//Generisano na osnovu sablona: service.ftl
//Element modela: GremoNaElektriko::Vozilo


package com.ftn.mbrs.service;

import java.util.List;

import com.ftn.mbrs.model.Vozilo;

public interface VoziloService {

	public Vozilo save(Vozilo vozilo, Long modelVozilaId, Long karticaId);
	
	public Vozilo update(Vozilo vozilo);

	public String delete(Long id);
	
	public Vozilo findOne(Long id);
	
	public List<Vozilo> findAll();
}

File: mbrs/src/main/resources/static/js/services/VoziloService.js
//Fri May 11 23:51:24 CEST 2018
//Generisano na osnovu sablona: jsservice.ftl
//Element modela: GremoNaElektriko::Vozilo

angular.module('gremoNaElektrikoApp.VoziloService', [])
	.factory('VoziloService', function($http) {
		var factory = {};
		
		factory.findAll = function() {
			return $http.get('/gremoNaElektriko/vozilo');
		}
		
		factory.createVozilo = function(vozilo , modelVozilaId, karticaId) {
			return $http.post('/gremoNaElektriko/vozilo'+'/'+modelVozilaId+'/'+karticaId, vozilo);
		}

		factory.deleteVozilo = function(id) {
			return $http.delete('/gremoNaElektriko/vozilo/'+id);
		}
		
		factory.updateVozilo = function(vozilo) {
			return $http.put('/gremoNaElektriko/vozilo', vozilo);
		}
		
		return factory;
	});